src/components/verificationForms/VerifyIn.js

```diff
@@ -29,38 +29,46 @@ export default function VerifyIn(props) {
 
   const handleSubmit = async (event) => {
     event.preventDefault();
-    console.log(inProfValues)
+    console.log(inProfValues);
     const res = await fetch("http://localhost:3000/api/indprofilecomp/", {
-      method: 'POST',
+      method: "POST",
       headers: {
-        'Content-Type': 'application/json'
+        "Content-Type": "application/json",
       },
       body: JSON.stringify({
-        inProfValues
-      })
-    })
-    const resp = await res.json()
-    if (resp.error || resp.success==false){
-     console.log(resp);
-    }else{
+        inProfValues,
+      }),
+    });
+    const resp = await res.json();
+    if (resp.error || resp.success == false) {
+      console.log(resp);
+    } else {
       console.log(resp);
-      console.log("Saved individual successfully")
-  }
+      console.log("Saved individual successfully");
+      //redirect to individual profile
+      Router.push(`/profile-individual-form/${id}`);
```

Comment:
Am i supposed to use /individual-dashboard here?

Reply:
As discussed in the meet, you just have to send the user to login page after profile completion and not their respective dashboards. Just add a redirect to the login page and it will be done.

Reply:
added /login here

Reply:
have you tested it?

Reply:
700 status 
<img width="376" alt="image" src="https://user-images.githubusercontent.com/60812924/152214700-4e44e1f4-0f42-4f79-afb3-a8a24154e58d.png">

Reply:
Rectify the error and make sure the redirect is working fine.

Reply:
i have verified ,its a nodemailer issue with my unverified email. Works fine on verified accounts
\